src/pages/index.js

```diff
@@ -37,44 +37,43 @@ const userInfo = new UserInfo(
   "#profile-des",
   "#profile-image"
 );
-let userID;
-api
-  .getUserinfo()
-  .then((data) => {
-    const { name, about, avatar, _id } = data;
-    userID = _id;
-    userInfo.setUserInfo(name, about);
-    userInfo.setUserProfileImage(avatar);
-  })
-  .catch((error) => {
-    console.error("Error fetching data:", error);
-  });
+
+let userId;
+let cardSection;
 
 // functions
 function handleProfileFormSubmit(data) {
   const { name, about } = data;
   api
     .updateProfile(name, about)
-    .then(() => {
-      profileEditPopup.editSubmitBtn();
-    })
+    .then(profileEditPopup.editSubmitBtn())
     .then(() => {
       userInfo.setUserInfo(name, about);
-      profileEditPopup.close();
-      profileEditPopup.resetSubmitBtn();
     })
-    .catch((error) => {
-      console.error("Error fetching data:", error);
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      profileEditPopup.resetSubmitBtn();
+      profileEditPopup.close();
     });
 }
 
 function handleCardFormSubmit(data) {
-  api.postCard(data).then((data) => {
-    const card = createCard(data);
-    cardSection.prependItem(card);
-    profileAddPopup.close();
-    profileAddPopup.resetSubmitBtn();
-  });
+  api
+    .postCard(data)
+    .then(profileAddPopup.editSubmitBtn())
```

Comment:
`.then(profileEditPopup.editSubmitBtn())`


please, delete it


it should be **before** the request